apps/web/src/components/forms/2fa/enable-authenticator-app-dialog.tsx

```diff
@@ -197,7 +197,14 @@ export const EnableAuthenticatorAppDialog = ({
                   />
 
                   <DialogFooter>
-                    <Button type="button" variant="secondary" onClick={() => onOpenChange(false)}>
+                    <Button
+                      type="button"
+                      variant="secondary"
+                      onClick={() => {
+                        onOpenChange(false);
+                        setupTwoFactorAuthenticationForm.setValue('password', '');
```

Comment:
Could we just call `setupTwoFactorAuthenticationForm.reset()`? I forget if there's other state we care about there upon cancelling.

Reply:
Alternatively I'd use effect the `open` state and reset the form there if we've closed it

Reply:
We do have a branch pushed with a fix for this, https://github.com/documenso/documenso/tree/chore/form-reset 😶